src/common/checksum.c

```diff
@@ -1352,6 +1372,82 @@ ic_idx2csum(struct dcs_iod_csums *iod_csum, uint32_t iod_idx,
 	return csum_info->cs_csum + offset;
 }
 
+daos_size_t
+ic_size(struct dcs_iod_csums *obj)
+{
+	daos_size_t	result = sizeof(*obj);
+	int		i;
+
+	for (i = 0; i < obj->ic_nr; i++)
+		result += ci_size(obj->ic_data[i]);
+
+	return result;
+}
+
+int
+ic_serialize(struct dcs_iod_csums *iod_csums, d_iov_t *iov)
```

Comment:
This is only used for test code?  then probably move to src/common/tests ?

Reply:
Actually, not needed at all anymore. Removed.